src/network/useQueryClient.js

```diff
@@ -0,0 +1,8 @@
+import { useQuery } from "@tanstack/react-query"
+import { getRequest } from "./axiosClient"
+import {urls} from './urls'
+
+export const useGetHandler = (key) => {
+    const response = useQuery({ queryKey: [key], queryFn: () => getRequest(urls.USERS)})
```

Comment:
This isn't generic anymore
```suggestion
export const useGetHandler = (key, url) => {
    const response = useQuery({ [key], () => getRequest(url)})
```